examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+let query=['INSERT INTO todos(id,done) VALUES '+mysql.escape(18,0),
+           'UPDATE todos SET name="ali",due="2019-08-31" WHERE id='+mysql.escape(5),
+           'DELETE FROM todos WHERE NAME='+mysql.escape("ali"),
```

Comment:
What if the todo item had the name "Ali", or "My name is ali" or "My name is Ali"? Will this query address them, or not?

Also, here you use `NAME` for the field which is called `Name` in the database. Earlier you used `name`. You also use `done` and `id` when the database fields are called `Done` and `Id`. I know this will run - it does work. But it could be a bit confusing for a reader.